addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24051,6 +24051,12 @@ msgstr ""
 
 #empty strings from id 39202 to 39999
 
+#. Subtitle line spacing setting
```

Comment:
I suppose it would make sense to put your new label above the `#empty strings from id ...` line and then also adjust the starting ID number accordingly, since you are now using this number :)

Reply:
Sorry, merge artifact. Fixed.